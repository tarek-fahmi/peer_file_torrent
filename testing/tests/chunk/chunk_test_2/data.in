Omit Invalid Chunks
./testing/bin/pkg_main ./testing/resources/invalid_1.bpkg -chunk_check
./testing/bin/pkg_main ./testing/resources/invalid_2.bpkg -chunk_check