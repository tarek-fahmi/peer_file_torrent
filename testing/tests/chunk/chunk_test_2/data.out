6359c145c0ba83b1c260d51c83c451fa160d09e865f1e54574116c5d5a8ae255
40ba19c4707044f18cfd5694d2e75abf15b325a93c12be7ae6fa6eec7a1feaf9
7c274a0ba71e34a66ef792d215b2bde988d60fce1c0065f269e8427278270091
b9d2572bd343fb0df1fd06291a376a71859868e371b4c94be07d362d00c4f563
622c61a5170d1854127edfed683c998b0a5ee623d48ba7b6c1c725574fcbabfa
e3e6546371160246310496b18c45c7e74904ce9062784879e53686f3e7465e13
f15a8caf6115a3c234f222b845a5dc3a8ea49daa45f70445a5a285efb54959e2
fb3b3a155b0854404b06601f02518d68b16b7dd5be6df9a43042140ad1a82287
c6665a3d9d865683d46779af0eacce17e67f7c19dd004dcde63b33cb56c0bc62
862c4f9f9596feb17c3d00602d83985296b8f98ff119828f58f772a799b3dd20
a8d9dd2fee94c3c53d5281dc726e1640a1dcdb631f3cfb1c7a620da47f140af3
3e9c976e4a19306b48d6a8d59b67110cfcd86511d8932357205cbd31382ce9fa
dcedbcd6e54f57a3a1cda150ae33dfb165816755f4ca1307b25005388bae7d3a
dcab6c1e8030dd2b7dd14019d9f75cf48c7e788620e010d1094dfc6fa6ef5718
890592a2e3e67dae344a746a0a9894836613cfebc2f793cb40363cf399a75da0
062e5f329bedff6d01d0657d1d170c884ffdb552c16d74c99d54bd743922b8d5